pkg/env/env.go

```diff
@@ -93,6 +93,7 @@ type Interface interface {
 	FPAuthorizer(string, ...string) (autorest.Authorizer, error)
 	FPNewClientCertificateCredential(string) (*azidentity.ClientCertificateCredential, error)
 	FPClientID() string
+	FPServicePrincipalID() string
```

Comment:
Don't need this anymore.  Also it varies per tenant.

Reply:
Removed the usage!